### ISSUE TITLE
yolo v3

### ISSUE THREAD

**Issue by ssetty** [experiencor/keras-yolo3#149]
(1) pretrained weights for backend : is it pretrained model based on coco dataset? 

(2) I want to understand termination condition usually for object detection we minimize average loss, maximize average iou, maximize mAP... 
but in this code I observe
avg_obj                 [14][0.762737632]
avg_noobj               [14][0.0172626134]
avg_iou                 [14][0.857628465]
avg_cat                 [14][0.999933302]
recall50        [14][0.799946666]
recall75        [14][0.733284414]
count   [14][15]

Training continues for more than 24 hours what is convergence criteria - what part of code does this? 

(3) Is this framework based on yolo-v3?

Thanks

**Comment by AbhishekP-Ai** [comment]
This part is being given by the yolo.py.
